### PR TITLE
Jagadeesh_Assignment_21_15+20min

### PR DESCRIPTION
**Pull request by JagadeeshMJun20**
Please review my code.
Output:
image

image

### CODE REVIEW

File: src/jagadeeshMJun20/FibonacciSeries.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+package jagadeeshMJun20;
+
+import java.util.Scanner;
+
+public class FibonacciSeries {
+	private int num;
+
+	void getInput(int number) {
+		num = number;
+
```
Comment (MadhuraMJun20) [inline src/jagadeeshMJun20/FibonacciSeries.java:23]: Remove all excess spaces.

File: src/jagadeeshMJun20/FibonacciSeries.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+the 2 is found by adding the two numbers before it (1+1),
+the 3 is found by adding the two numbers before it (1+2),
+the 5 is (2+3),
+and so on! */
+package jagadeeshMJun20;
+
+import java.util.Scanner;
+
+public class FibonacciSeries {
+	private int num;
```
Comment (MadhuraMJun20) [inline src/jagadeeshMJun20/FibonacciSeries.java:19]: Good job using private variable and method.

File: src/jagadeeshMJun20/FibonacciSeries.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+	void getInput(int number) {
+		num = number;
+
+		displayOutput();
+	}
+
+	private void displayOutput() {
```
Comment (MadhuraMJun20) [inline src/jagadeeshMJun20/FibonacciSeries.java:27]: This method works good. However I think there can be a simpler way for doing this without using an array --
for (int index = 0; index < n; index++) {
			int sum = firstNum + nextNum;
			System.out.print(nextNum + " ");
			firstNum = nextNum;
			nextNum = sum;
		}

File: src/jagadeeshMJun20/SumNumbers.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+		number2 = num2;
+		displaySum();
+	}
+
+	private void displaySum() {
+		number1 = number1 + number2;
+		number2 = number1 - number2;
+		number1 = number1 - number2;
+		System.out.println("Output: " + number1 + ", " + number2);
+
```
Comment (MadhuraMJun20) [inline src/jagadeeshMJun20/SumNumbers.java:23]: Perfect logic!

**Review by MadhuraMJun20 — APPROVED** [review]

File: src/jagadeeshMJun20/FibonacciSeries.java (hunk 2 above)
Comment (JagadeeshMJun20) [reply src/jagadeeshMJun20/FibonacciSeries.java:19]: Yes, concept of encapsulation.

File: src/jagadeeshMJun20/FibonacciSeries.java (hunk 1 above)
Comment (JagadeeshMJun20) [reply src/jagadeeshMJun20/FibonacciSeries.java:23]: I have removed all spaces.

**Pull request merged**